app/views/content/train-to-be-a-teacher/assessment-only-route-to-qts.md

```diff
@@ -9,6 +9,10 @@ image: false
 backlink: /
 promo_content:
     - content/train-to-be-a-teacher/promos/adviser-promo-assessment-only
+cta_adviser:
```

Comment:
@martyn-w We need to remove from this assessment only page before it is merged to prod

Would it be worth replacing an adviser promo on a page where we actually want the new promo?

Eg `/train-to-be-a-teacher/if-you-have-a-degree` (would need to use the bespoke wording)